# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1345336732 14400
#      Sat Aug 18 20:38:52 2012 -0400
# Node ID d0715b4302861ad5c41222e3bd0a3d5a3525675a
# Parent  c3d463c9360456599c57281bdb413c7bbd8eac4d
fix gui sources missing from dist

* libgui/Makefile.am (EXTRA_DIST): New variable.
* libgui/src/Makefile.am (octave_gui_ICONS): New variable.
(EXTRA_DIST): Include $(ocxtave_gui_ICONS) in the list.

diff --git a/libgui/Makefile.am b/libgui/Makefile.am
--- a/libgui/Makefile.am
+++ b/libgui/Makefile.am
@@ -1,3 +1,5 @@
 SUBDIRS = src
 
 octetc_DATA = default-qt-settings
+
+EXTRA_DIST = default-qt-settings
diff --git a/libgui/src/Makefile.am b/libgui/src/Makefile.am
--- a/libgui/src/Makefile.am
+++ b/libgui/src/Makefile.am
@@ -117,16 +117,17 @@ noinst_HEADERS = \
   m-editor/find-dialog.h \
   m-editor/lexer-octave-gui.h \
   main-window.h \
   octave-adapter/octave-event-listener.h \
   octave-adapter/octave-event-observer.h \
   octave-adapter/octave-event.h \
   octave-adapter/octave-link.h \
   octave-adapter/octave-main-thread.h \
+  octave-gui.h \
   octave-qt-event-listener.h \
   qtinfo/parser.h \
   qtinfo/webinfo.h \
   resource-manager.h \
   settings-dialog.h \
   symbol-information.h \
   terminal-dockwidget.h \
   welcome-wizard.h \
@@ -137,17 +138,59 @@ CLEANFILES = $(octave_gui_MOC) $(octave_
 
 clean-local:
 	-if test "$(srcdir)" != "."; then \
 		for d in $(SRCDIRS); do test -d $$d && rmdir $$d; done \
 	 fi
 
 BUILT_SOURCES = $(octave_gui_UI_H)
 
-EXTRA_DIST = $(octave_gui_UI) resource.qrc
+octave_gui_ICONS = \
+  icons/arrow_right.png \
+  icons/artsbuilderexecute.png \
+  icons/bookmark.png \
+  icons/bp_next.png \
+  icons/bp_prev.png \
+  icons/bp_rm_all.png \
+  icons/bp_toggle.png \
+  icons/chat.png \
+  icons/configure.png \
+  icons/db_cont.png \
+  icons/db_step_in.png \
+  icons/db_step_out.png \
+  icons/db_step.png \
+  icons/db_stop.png \
+  icons/editcopy.png \
+  icons/editcut.png \
+  icons/editpaste.png \
+  icons/filenew.png \
+  icons/fileopen.png \
+  icons/filesaveas.png \
+  icons/filesave.png \
+  icons/find.png \
+  icons/help_index.png \
+  icons/icons_license \
+  icons/jabber_protocol.png \
+  icons/logo.png \
+  icons/question.png \
+  icons/redled.png \
+  icons/redo.png \
+  icons/search.png \
+  icons/star.png \
+  icons/stop.png \
+  icons/terminal.png \
+  icons/undo.png \
+  icons/up.png \
+  icons/zoom-in.png \
+  icons/zoom-out.png
+
+EXTRA_DIST = \
+  $(octave_gui_UI) \
+  $(octave_gui_ICONS) \
+  resource.qrc
 
 noinst_LTLIBRARIES = libqterminal.la
 
 noinst_HEADERS += ../qterminal/libqterminal/QTerminal.h \
 		  ../qterminal/libqterminal/QTerminalInterface.h \
 		  ../qterminal/libqterminal/win32/QTerminalColors.h \
 		  ../qterminal/libqterminal/win32/QWinTerminalImpl.h \
 		  ../qterminal/libqterminal/unix/BlockArray.h \
